docs/storage-overview.md

```diff
@@ -15,7 +15,7 @@ The basis needs of Genestack are the following storage classes
 | Storage Type | Description |
 |--------------|-------------|
 | general | A general storage cluster which is set as the deault |
-| general-multi-attach | A multi-read/write storage backend |
+| general-multi-attach | A multi-read/write storage backend mainly required for glance |
```

Comment:
Not a bad note, but multi-attached storage can be used for purposes other than glance. This is just a description of storage types used, we should move the multi-attached storage notes to the glance docs.